src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt

```diff
@@ -31,7 +31,8 @@ class LootOneNearbyPokestop(val sortedPokestops: List<Pokestop>, val lootTimeout
 
         if (nearbyPokestops.size > 0) {
             val closest = nearbyPokestops.first()
-            Log.normal("Looting nearby pokestop ${closest.id}")
+            val pokestopName = nearbyPokestops.first().details.name
```

Comment:
Sadly, I don't know how to do that :( 
This was the best thing I could do with my knowledge.

If anyone could take this ideea and make like @jabbink said, that would be great